README.md

```diff
@@ -833,6 +833,26 @@ when you need to retrieve a single record by some attributes.
   Mitigate code duplication by using partial templates and layouts.
 <sup>[[link](#partials)]</sup>
 
+* <a name="no-instance-variables-in-partials"></a>
+  Avoid using instance variables in partials, pass a local variable to `render` instead. Partial may be used
+  in a different controller or action, where the variable can have a different name or be absent at all. In
+  these cases undefined instance variable will not raise an exception whereas local variable will. 
+<sup>[[link](#no-instance-variables-in-partials)]</sup>
+
+  ```ruby
+  # app/views/courses/show.html.erb
+  # bad
+  <%= render 'course_description' %>
+  # good
+  <%= render 'course_description', course: @course %>
+  
+  # app/views/courses/_course_description.html.erb
+  # bad
+  <%= @course.description %>
+  # good
+  <%= course.description %>
+  ```
```

Comment:
I think the meaning would be clearer if the entire bad example was in one listing, and the good one as well:
```erb
# bad
# app/views/courses/show.html.erb
<%= render 'course_description' %>
# app/views/courses/_course_description.html.erb
<%= @course.description %>

# good
# app/views/courses/show.html.erb
<%= render 'course_description', course: @course %>
# app/views/courses/_course_description.html.erb
<%= course.description %>
```

Reply:
They say it's better late than never! Updated.